.tekton/operator-bundle-pipeline.yaml

```diff
@@ -658,3 +661,92 @@ spec:
       - name: kind
         value: task
       resolver: bundles
+
+  - name: create-acs-style-snapshot
+    # Explicitly running after all other tasks to ensure that
+    # - there are no failures that should prevent a release of the operator-bundle image (missing RPMs signatures, deprecated base images, ...)
+    # - the source image is present as it is required by EC
+    runAfter:
+    - apply-tags
+    - build-source-image
+    - clair-scan
+    - clamav-scan
+    - deprecated-base-image-check
+    - rpms-signature-scan
+    - sast-snyk-check
+    - push-dockerfile
+    params:
+    - name: SOURCE_ARTIFACT
+      value: $(tasks.clone-repository.results.SOURCE_ARTIFACT)
+    - name: IMAGE_REFS
+      value: |
+        [
+          {
+            "component": "central-db",
+            "ref": "$(params.central-db-image-build-repo)@$(tasks.wait-for-central-db-image.results.IMAGE_DIGEST)",
+            "repository": "$(params.git-url)",
+            "revision": "$(params.revision)"
+          },
+          {
+            "component": "collector",
+            "ref": "$(params.collector-full-image-build-repo)@$(tasks.wait-for-collector-full-image.results.IMAGE_DIGEST)",
+            "repository": "$(tasks.wait-for-collector-full-image.results.GIT_REPO)",
+            "revision": "$(tasks.wait-for-collector-full-image.results.GIT_REF)"
+          },
+          {
+            "component": "main",
+            "ref": "$(params.main-image-build-repo)@$(tasks.wait-for-main-image.results.IMAGE_DIGEST)",
+            "repository": "$(params.git-url)",
+            "revision": "$(params.revision)"
+          },
+          {
+            "component": "operator",
+            "ref": "$(params.operator-image-build-repo)@$(tasks.wait-for-operator-image.results.IMAGE_DIGEST)",
+            "repository": "$(params.git-url)",
+            "revision": "$(params.revision)"
+          },
+          {
+            "component": "operator-bundle",
+            "ref": "$(tasks.build-container.results.IMAGE_URL)@$(tasks.build-container.results.IMAGE_DIGEST)",
```

Comment:
`$(tasks.build-container.results.IMAGE_URL)` will include tag. You may want to use `output-image-repo` for the `operator-bundle` for consistency with others, or on the contrary, add tags to others. Or not do anything. Up to you.

Reply:
Done `output-image-repo` in https://github.com/stackrox/stackrox/pull/13577/commits/aa031ef6d3e4cedc0d2ab3e8016caa113d3de7ef